src/services/video-conference-manager/index.ts

```diff
@@ -293,6 +300,10 @@ export default class VideoConfereceManager {
     this.messageBridge.publish(MeetingEvent.FRAME_PARENT_SIZE_UPDATE, { height, width });
   };
 
+  private updateFrameLocale = (): void => {
+    this.messageBridge.publish(MeetingEvent.FRAME_LOCALE_UPDATE, this.frameLocale);
+  };
```

Comment:
you will need to create two validations:

1 - If there is a list of languages to be added.
2 - If the default language selected is present in the list of languages

this is just a draft, you can create a validation to test this.

```suggestion
  private updateFrameLocale = (): void => {
  const { language, locales } = this.frameLocale
  
   if(!locales.length || !language) return;
   const languages = this.frameLocale.locales.map(locale => locale.language)
   const avaliableLanguages = ['pt-Br', 'en', ...languages];
   
    if(avaliableLanguages.includes(language))  {
     throw new Error('the default language is not available in the language listing')
    }

  
    this.messageBridge.publish(MeetingEvent.FRAME_LOCALE_UPDATE, this.frameLocale);
  };
```